.github/workflows/close-stale-issues.yml

```diff
@@ -29,9 +31,9 @@ jobs:
 
         # These labels are required
         stale-issue-label: blocked/close-if-inactive
-        exempt-issue-labels: no-autoclose, stage/needs-attention
+        exempt-issue-labels: no-autoclose, stage/needs-attention, type/feature, stage/needs-triage, type/bug, stage/needs-investigation 
```

Comment:
Should we include;
- [maintainer/need-followup](https://github.com/aws/aws-sam-cli/labels/maintainer%2Fneed-followup)
- [maintainer/need-response](https://github.com/aws/aws-sam-cli/labels/maintainer%2Fneed-response)